### PR TITLE
feat/FWS-28/Ability_to_create_a_user

### PR DESCRIPTION
**Pull request by Eli293**
JIRA: link to jira ticket
https://champlainsaintlambert.atlassian.net/browse/FWS-28
## Context:
This ticket is about the ability to sign in and create a new member that will be added into our backend database.
## Changes

- Added a Sign Up button near the Log In
- The Sign up will directly bring you to sign up page
- Additional Username Text field
- Now adds the member's info inside the database after creation 
- This affects how state will be handled throughout the whole web application

## UI Changes:

NEW
image

NEW
image

### CODE REVIEW

**Review by Abdur-Siddiqui — APPROVED** [review]
Lookgs good good job

**Review by emilegirardGit — APPROVED** [review]
Everything looks good

File: src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerUnitTest.java [hunk 1]
```diff
@@ -1,16 +1,59 @@
...
     @Autowired
     WebTestClient webTestClient;
     @MockBean
     AccountService accountService;
-}
\ No newline at end of file
+    @InjectMocks
+    private AccountController accountController;
+    //THE ID BELOW WORKS ON EMILE'S COMPUTER WILL HAVE TO FIX W/E IS WRONG WITH MINE
+    private String testToken = "Bearer eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjlBNEZQeXE4LTh1OVZqR1IycmJQTiJ9..h---_3EE-0gQ6oKpSQm1-85_icKtLH40u8S6GR9stSCBoBwI7tuaHcX0TJ47QKvRGl4zqY9LxvCrEJ392MFAVn0te-BOmUQgEqzG9WiYz-87psSx8Q1GDPR8kDnmBx6fPTwexxTWPUPgc3dyeR2j5BGfjl4qJp0hk0ZH1h61FLk75Cx-ojFis2UVWbswmPqIj_o0bstxCwle4UDoyr5PIb88vE0_BTmx1WtW-kjZguzcWhXX7U1aB9fD18Qw7g";
```
Comment (AlexeiTimbro) [inline src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerUnitTest.java:26]: WHy did implement the test token into your test it looks like a smart thing to do?

Comment (Eli293) [reply src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerUnitTest.java:26]: Good question! We need this test token in the header to test the API calls while having the right authorization to do so.

**Review by AlexeiTimbro — APPROVED** [review]
Good job Elias.

**Pull request merged**